pkg/pillar/zboot/zboot.go

```diff
@@ -382,7 +382,7 @@ func MarkCurrentPartitionStateActive() error {
 
 // XXX known pathnames for the version file and the zededa-tools container
 const (
-	newVersionFile = "/etc/eve-release"
+	newVersionFile = "/run/eve-release"
 	// XXX remove use of shortVersionFile in container once the deployed
 	// images all have newVersionFile
 	shortVersionFile = "/opt/zededa/bin/versioninfo"
```

Comment:
While you're at it, can we also kill this legacy shortVersionFile @gkodali-zededa ? There's literally not a slimmest chance we would have devices without /etc/eve-release in any of the partition.

This should also be accompanied by removing it from images/*.yml

Reply:
will do

Reply:
Done

Reply:
@rvs +1. Its been eons since we did the go-provision to eve transition when we had to support the an other image without /etc/eve-release